ios/Framework/Core/Extensions/String+Screens.swift

```diff
@@ -38,4 +43,27 @@ extension String {
 		return Int64(self)
 	}
 
+	public func asLocalized(locale: NSLocale) -> String {
```

Comment:
I'm going to use the same method for Android. It's more elegant than what I was trying.